cpplint/README

```diff
@@ -1,5 +1,5 @@
 This is automated checker to make sure a C++ file follows Google's C++ style
-guide (http://google.github.io/styleguide/cppguide.html). As it
+guide [https://google.github.io/styleguide/cppguide.html](https://google.github.io/styleguide/cppguide.html). As it
```

Comment:
This doesn't seem to be a Markdown file, so I don't think it should have Markdown-style links.